metsim/disaggregate.py

```diff
@@ -75,17 +75,21 @@ def temp(df_daily, df_disagg, t_t_min, t_t_max, ts):
     temp = np.append(np.insert(temp, 0, temp[0:2]), temp[-2:])
 
     # Interpolate the values
-    interp = scipy.interpolate.PchipInterpolator(time, temp, extrapolate=True)
-    temps = interp(ts * np.arange(0, len(df_disagg.index)))
+    try:
+        interp = scipy.interpolate.PchipInterpolator(time, temp, extrapolate=True)
+        temps = interp(ts * np.arange(0, len(df_disagg.index)))
+    except ValueError:
+        temps = np.full(len(df_disagg.index), np.nan)
+
```

Comment:
I am confused by this anyway - why would we even be handling ocean cells if they don't have associated data. I think I am basically commenting on the same thing as @jhamman . We should only run over the 'active' grid cells, so the interpolator should never be called on ocean grid cells (or cells outside the active mask). This is important for the parallelism as well, because you don't want large parts that are outside the mask be assigned to a process that then just sits idle.